### PR TITLE
Announce upcoming 1.1 EOL.

### PR DESCRIPTION
**Pull request by duderino**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[x] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2019/announcing-1.1-eol/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Support for Istio 1.1 ends on September 19th, 2019
+description: Upcoming Istio 1.1 end of life announcement.
+publishdate: 2019-08-15
+attribution: The Istio Team
+release: 1.1
```
Comment (geeknoid) [inline content/en/blog/2019/announcing-1.1-eol/index.md]: Nuke this, it's only useful for release note posts.

**Comment by geeknoid** [comment]
Given the directory shuffle I just did, I'm not sure if the cherry picker will handle things correctly...

File: content/en/blog/2019/announcing-1.1-eol/index.md (hunk 1 above)
Comment (geeknoid) [inline content/en/blog/2019/announcing-1.1-eol/index.md]:
```suggestion
release: 1.1
```

Comment (duderino) [reply content/en/blog/2019/announcing-1.1-eol/index.md]: I don't see the change?

**Comment by duderino** [comment]
> Given the directory shuffle I just did, I'm not sure if the cherry picker will handle things correctly...

I'll send a PR manually if needed

File: content/en/blog/2019/announcing-1.1-eol/index.md (hunk 1 above)
Comment (geeknoid) [reply content/en/blog/2019/announcing-1.1-eol/index.md]: Delete "release: 1.1". That kind of annotation is only for release notes.

**Pull request merged**